@kiva/kv-loan-filters/src/loanChannelQueryMap.ts

```diff
@@ -1063,8 +1069,381 @@ export default {
078,15081,15082,15083,15084,15085,15086,15087,15088,15089,15090,15091,15095,15096,15101,15102,15104,15106,15108,15110,15112,15116,15120,15122,15123,15126,15127,15129,15131,15132,15133,15134,15135,15136,15137,15139,15140,15142,15143,15144,15145,15146,15147,15148,15201,15202,15203,15204,15205,15206,15207,15208,15209,15210,15211,15212,15213,15214,15215,15216,15217,15218,15219,15220,15221,15222,15223,15224,15225,15226,15227,15228,15229,15230,15231,15232,15233,15234,15235,15236,15237,15238,15239,15240,15241,15242,15243,15244,15250,15251,15252,15253,15254,15255,15257,15258,15259,15260,15261,15262,15263,15264,15265,15267,15268,15270,15272,15274,15275,15276,15277,15278,15279,15281,15282,15283,15285,15286,15289,15290,15295,15301,15310,15311,15312,15313,15314,15315,15316,15317,15320,15321,15322,15323,15324,15325,15327,15329,15330,15331,15332,15333,15334,15336,15337,15338,15339,15340,15341,15342,15344,15345,15346,15347,15348,15349,15350,15351,15352,15353,15354,15357,15358,15359,15360,15361,15362,15363,15364,15365,15366,15367,15368,15370,15376,15377,15378,15380,15401,15410,15413,15415,15416,15417,15420,15421,15422,15425,15428,15430,15431,15433,15435,15436,15437,15438,15439,15440,15442,15443,15444,15445,15446,15447,15448,15449,15450,15451,15454,15455,15456,15458,15459,15460,15461,15462,15463,15464,15465,15466,15467,15468,15469,15470,15472,15473,15474,15475,15476,15478,15479,15480,15482,15484,15486,15488,15489,15490,15492,15601,15610,15611,15612,15613,15615,15616,15617,15618,15619,15620,15621,15622,15623,15624,15625,15626,15627,15628,15629,15630,15631,15632,15633,15634,15635,15636,15637,15638,15639,15640,15641,15642,15644,15646,15647,15650,15655,15656,15658,15660,15661,15662,15663,15664,15665,15666,15668,15670,15671,15672,15673,15674,15675,15676,15677,15678,15679,15680,15681,15682,15683,15684,15685,15686,15687,15688,15689,15690,15691,15692,15693,15695,15696,15697,15698,15701,15710,15713,15716,15717,15720,15723,15725,15727,15731,15732,15736,15739,15747,15750,15751,15752,15754,15756,15763,15772,15774,15779,15783,15901,15902,15904,15905,15906,15907,15909,15915,15920,15923,15929,15931,15943,15944,15945,15948,15949,15951,15952,15954,15956,15958,16001,16002,16003,16020,16021,16022,16023,16024,16025,16027,16029,16030,16033,16034,16035,16037,16038,16039,16040,16041,16045,16046,16048,16050,16051,16052,16053,16055,16056,16057,16059,16061,16063,16066,16101,16102,16103,16105,16107,16108,16110,16111,16112,16113,16114,16115,16116,16117,16120,16121,16123,16124,16125,16127,16130,16131,16132,16133,16134,16137,16140,16142,16143,16145,16146,16148,16150,16151,16153,16154,16155,16156,16157,16159,16160,16161,16201,16210,16211,16212,16216,16218,16222,16223,16226,16228,16229,16236,16238,16244,16246,16256,16311,16314,16316,16327,16328,16335,16354,16360,16388,16401,16403,16404,16406,16407,16410,16411,16412,16413,16415,16417,16421,16422,16423,16424,16426,16427,16428,16430,16432,16433,16434,16435,16438,16440,16441,16442,16443,43952,43953,15001,15003,16675,16699&distributionModel=direct',
```

Comment:
Noting we should be able to query through with `postalCode` for flss now too... haven't tested it much and it's only a pass through filter for now. Don't think we have any public facing interfaces for it

Reply:
Yup, but with having the city filter now too, I kept it simpler with city.